README.MD

```diff
@@ -95,64 +123,60 @@ $this->get('mongo.connection.{connectionName}'); // [test_db, other_db, test_db_
 
 To manipulate the database, please read the official [documentation](http://mongodb.github.io/mongo-php-library/classes/database/)
 
-### Fixtures
+## Query Profiling
+
+On dev environment all queries executed by the library MongoDB\Collection class are profiled and showed inside the symfony web profiler.
+
+[![Profiler Toolbar](https://github.com/facile-it/mongodb-bundle/blob/master/docs/img/profiler_toolbar.png)](https://github.com/facile-it/mongodb-bundle/blob/master/docs/img/profiler_toolbar.png)
+
+[![Profiler Panel](https://github.com/facile-it/mongodb-bundle/blob/master/docs/img/profiler_panel.png)](https://github.com/facile-it/mongodb-bundle/blob/master/docs/img/profiler_panel.png)
+
+## Fixtures
 This bundle supports doctrine style fixtures.
-To create one you will need to add a php class in your bundle's `/DataFixtures/Mongo` directory. 
+To create one you will need to add a php class in your bundle's `/DataFixtures/Mongo` directory.
```

Comment:
I didn't have to do this in my project... I do `bin/console mongodb:fixtures:load tests/Fixtures/Mongo`

Reply:
@Jean85 Thanks. Fixed and restructured the whole section: https://github.com/facile-it/mongodb-bundle/tree/2105_%23117_sync_doc#fixtures
Is it clear?